demos/real_time_stream_analysis/python/README.md

```diff
@@ -101,29 +135,26 @@ ffmpeg -stream_loop -1 -i ./video.mp4 -f rtsp -rtsp_transport tcp rtsp://localho
 ffmpeg -f dshow -i video="HP HD Camera" -f rtsp -rtsp_transport tcp rtsp://localhost:8080/channel1
 ```
 
+While the RTSP stream is active, run the client to read it and send the output stream 
 ```bash
 python3 client.py --grpc_address localhost:9000 --input_stream 'rtsp://localhost:8080/channel1' --output_stream 'rtsp://localhost:8080/channel2'
 ```
 
-Then read rtsp stream using ffplay
+The results can be examined with ffplay utility which reads and display the altered content.
 
 ```bash
 ffplay -pixel_format yuv420p -video_size 704x704 -rtsp_transport tcp rtsp://localhost:8080/channel2
 ```
 
-### Inference using prerecorded video
 
-One might as well use prerecorded video and schedule it for inference.
-Replace video.mp4 with your video file.
 
-```bash
-python3 client.py --grpc_address localhost:9000 --input_stream 'workspace/video.mp4' --output_stream 'workspace/output.mp4'
-```
+## gRPC calls with unary calls
 
-### Inference using webcam
+The helper class `StreamClient` supports only the calls using unary gRPC calls. In that case it should be initialized with a parameter `streaming_api=True`.
```

Comment:
Is that correct? I see `StreamClient` along with `unary gRPC calls` and then we have `streaming_api=True` 
What is it then? Unary or streaming?